src/AppBundle/Controller/DefaultController.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Please remove `src/AppBundle` directory at all, and also remove it from `AppKernel`